dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -22,21 +22,17 @@ def index
       ROLES.each do |role|
         apps = get_applications_by_role(role, include_associations: false)
 
-        apps_statuses = apps.select(:status, "count(id) AS total").group(:status)
-
         if regional_partner_value == REGIONAL_PARTNERS_NONE
-          apps_statuses = apps_statuses.where(regional_partner_id: nil)
+          apps = apps.where(regional_partner_id: nil)
         elsif regional_partner_value && regional_partner_value != REGIONAL_PARTNERS_ALL
-          apps_statuses = apps_statuses.where(regional_partner_id: regional_partner_value)
+          apps = apps.where(regional_partner_id: regional_partner_value)
         end
 
-        apps_statuses.group(:status).each do |group|
-          application_data[role][group.status] = {total: group.total}
+        #apps_statuses = apps.select(:status, "count(id) AS total").group(:status)
```

Comment:
Can this commented line be removed or is this being saved for something?

Reply:
🤦‍♀️